test/index.js

```diff
@@ -148,20 +154,22 @@ describe('<ReactQuill />', function() {
       value: value,
     });
     const quill = getQuillInstance(wrapper);
-    expect(wrapper.getNode().getEditorContents()).to.equal(value);
+    // @ts-ignore untyped instance
+    expect(wrapper.instance().getEditorContents()).to.equal(value);
   })
 
   it('uses a custom editing area if provided', () => {
-    const editingArea = DOM.div({id:'venus'});
+    const div = React.createFactory('div');
+    const editingArea = div({id:'venus'});
     const wrapper = mountReactQuill({}, editingArea);
     const quill = getQuillInstance(wrapper);
     expect(wrapper.getDOMNode().querySelector('div#venus')).not.to.be.null;
   })
 
   /**
-   * This can't be tested with the current state of JSDOM. 
-   * The selection functions have been shimmed in this test suite, 
-   * but they  will not work until DOM traversal is implemented in 
+   * This can't be tested with the current state of JSDOM.
```

Comment:
https://github.com/jsdom/jsdom/pull/2719

this might actually work now, worth taking a look